### PR TITLE
podman-*: add page

### PR DESCRIPTION
**Pull request by FearTheBadger**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/podman-images.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# podman images
+
+> Manage Podman images.
+> More information: <https://docs.podman.com/engine/reference/commandline/images/>.
```
Comment (blueskyson) [inline pages/common/podman-images.md:4]: https://docs.podman.com is unreachable now.
```suggestion
> More information: <https://docs.podman.io/en/latest/markdown/podman-images.1.html>.
```

**Comment by navarroaxel** [comment]
oh we have a conflict with #8849 here

File: pages/common/podman-images.md (hunk 1 above)
Comment (FearTheBadger) [reply pages/common/podman-images.md]: Good catch. I fixed all of the docs. Search and replace was my downfall.

File: pages/common/podman-rmi.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# podman rmi
+
+> Remove one or more Podman images.
+> More information: <https://docs.podman.io/en/latest/markdown/podman-rmi.1.html>.
+
+- Show help:
```
Comment (kbdharun) [inline pages/common/podman-rmi.md:6]:
```suggestion
- Display help:
```
Move this command to last

File: pages/common/podman-rmi.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`podman rmi`
+
+- Remove one or more images given their names:
+
+`podman rmi {{image:tag}} {{image2:tag}} {{...}}`
```
Comment (kbdharun) [inline pages/common/podman-rmi.md:12]:
```suggestion
`podman rmi {{image1 image2 ...}}`
```
`docker rmi` follows this too.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by blueskyson — APPROVED** [review]

File: pages/common/podman-rmi.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Force remove an image:
+
+`podman rmi --force {{image}}`
+
+- Remove an image without deleting untagged parents:
+
+`podman rmi --no-prune {{image}}`
```
Comment (kbdharun) [inline pages/common/podman-rmi.md:20]:
```suggestion
# podman rmi

> Remove one or more Podman images.
> More information: <https://docs.podman.io/en/latest/markdown/podman-rmi.1.html>.


- Remove one or more images given their names:

`podman rmi {{{image1 image2 ...}}`

- Force remove an image:

`podman rmi --force {{image}}`

- Remove an image without deleting untagged parents:

`podman rmi --no-prune {{image}}`

- Show help:

`podman rmi`
```
I made my previous suggestion simple to commit with this comment @FearTheBadger.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**